public/angular/js/services/admin_pwd.js

```diff
@@ -0,0 +1,21 @@
+angular.module('Aggie')
+
+.factory('admin_pwd',['$state', '$modal', '$http', 'FlashService',
```

Comment:
By convention, service names should be CapitalCamelCase. Please try to adhere to conventions in the app and in Angular more generally by checking other similar code. I appreciate it.